memory-viz/src/memory_model.ts

```diff
@@ -927,27 +940,27 @@ export class MemoryModel {
             obj.style = { ...obj.style, ...this.roughjs_config?.options };
 
             const frame_types = [".frame", ".blank-frame"];
-            if (frame_types.includes(obj.type) || obj.type === ".class") {
-                let is_frame = frame_types.includes(obj.type);
+            if (frame_types.includes(obj.type!) || obj.type === ".class") {
+                let is_frame = frame_types.includes(obj.type!);
 
                 const size = this.drawClass(
-                    obj.x,
-                    obj.y,
-                    obj.name,
-                    obj.id,
+                    obj.x!,
+                    obj.y!,
+                    obj.name!,
```

Comment:
Allow this to be undefined/null

Reply:
All changes are good, except I think you missed this comment. the `name` parameter for `drawClass` should allow for `undefined`/`null`